### PR TITLE
Escape backtick in relnotes' markdown

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
Before submitting a PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it)
* [x] Squashed related commits together.
* [ ] Added tests.
* [ ] Added an entry to the [Changelog](CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../CONTRIBUTING.md#changelog-entry-format).
* [ ] All tests are passing.
* [ ] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

So that syntax highlighting doesn't get all messed up.

### CODE REVIEW

File: relnotes/v0.38.0.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ### Bug fixes
 
-* Add `require 'time'` to `remote_config.rb` to avoid "undifined method `rfc2822'". ([@necojackarc][])
+* Add `require 'time'` to `remote_config.rb` to avoid "undifined method \`rfc2822'". ([@necojackarc][])
```
Comment (bbatsov) [inline relnotes/v0.38.0.md]: Actually I think rfc2822 should be in simple backticks and the rest was a mistake.

Comment (bbatsov) [inline relnotes/v0.38.0.md]: Ah, no - you're right.

Comment (deivid-rodriguez) [inline relnotes/v0.38.0.md]: You mean I should replace the closing `'` with another backtick instead? This "open with `` ` ``, close with `'` " style is very common in Ruby errors, so I'd say the original is the correct error message.

Comment (deivid-rodriguez) [inline relnotes/v0.38.0.md]: Ah, you had already replied, sorry. :)

**Pull request merged**